src/clients/RTorrent.ts

```diff
@@ -281,6 +326,61 @@ export default class RTorrent implements TorrentClient {
 			.mapErr((error) => (error === "FAILURE" ? "UNKNOWN_ERROR" : error));
 	}
 
+	async getAllDownloadDirs(options: {
+		metas: SearcheeWithInfoHash[] | Metafile[];
```

Comment:
looks like this could just take `infoHashes` instead of `metas`?

Reply:
This is an interface function and qbit needs `Searchee | Metafile` in order to handle subfolder layout.